guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,6 +4,12 @@
 [id="katello-headline-features"]
 == Headline Features
 
+*Inter Server Synchronization
+**Add the ability to do incremental syncable exports
+*Content Views
+**Allow hosts to register to multiple content view environments using subscription-manager
+*Over 70 bug fixes
+
 [id="katello-upgrade-warnings"]
```

Comment:
If there are no upgrade warnings / deprecations it's best to remove the chapters.